### PR TITLE
Created an endpoint to get a RSVP.

### PR DESCRIPTION
**Pull request by noshin7**

### CODE REVIEW

File: handlers/events.go [hunk 1]
```diff
@@ -52,3 +52,28 @@ func GetEvent(c *gin.Context) {
 	}
 	c.JSON(201, events) //success
 }
+
+func GetRSVP(c *gin.Context) {
+	var rsvp1 []models.GetRSVP
```
Comment (dianaabishop) [inline handlers/events.go:57]: maybe could name this something like rsvpList so it's a little clear about how it's different from the other RSVP var?

File: handlers/events.go [hunk 2]
```diff
@@ -52,3 +52,28 @@ func GetEvent(c *gin.Context) {
...
+
+	see_row := c.Param("responseID")
```
Comment (dianaabishop) [inline handlers/events.go:59]: totally a nit, but Go prefers camel case (seeRow) to snake case (see_row) for variables!

File: handlers/events.go [hunk 3]
```diff
@@ -52,3 +52,28 @@ func GetEvent(c *gin.Context) {
...
+	RSVProw, err := dbmap.Query(
+		"SELECT response_id, event_id, name, rsvp, comment_id FROM RSVP WHERE response_id=?;",
+		see_row)
+
+	fmt.Printf("%+v", RSVProw)
```
Comment (dianaabishop) [inline handlers/events.go:64]: stray print statement that you can just delete 🎉

File: handlers/events.go [hunk 4]
```diff
@@ -52,3 +52,28 @@ func GetEvent(c *gin.Context) {
...
+
+for RSVProw.Next() {
+	var rsvp models.GetRSVP
+	// for each row, scan into the event struct
+	err = RSVProw.Scan(&rsvp.ResponseID, &rsvp.EventID, &rsvp.Name, &rsvp.RSVP, &rsvp.CommentID)
+	if err != nil {
+	fmt.Println(err)
```
Comment (dianaabishop) [inline handlers/events.go:71]: stray Println here as well! All good to delete 😄

File: handlers/events.go [hunk 5]
```diff
@@ -52,3 +52,28 @@ func GetEvent(c *gin.Context) {
...
+		"SELECT response_id, event_id, name, rsvp, comment_id FROM RSVP WHERE response_id=?;",
+		see_row)
+
+	fmt.Printf("%+v", RSVProw)
+
+for RSVProw.Next() {
+	var rsvp models.GetRSVP
+	// for each row, scan into the event struct
+	err = RSVProw.Scan(&rsvp.ResponseID, &rsvp.EventID, &rsvp.Name, &rsvp.RSVP, &rsvp.CommentID)
+	if err != nil {
```
Comment (dianaabishop) [inline handlers/events.go:70]: I think the indenting is a bit wonky on this - an easy way to fix it is to `cd handlers` and then run `go fmt` and it'll format everything for you!

File: handlers/events.go (hunk 1 above)
Comment (dianaabishop) [reply handlers/events.go:57]: and you'll have to update the references to this var down below if you change the name!

**Review by dianaabishop — APPROVED** [review]
overall looking sensational!!! 🎉  Just some formatting and stylistic suggestions!

**Comment by noshin7** [comment]
INSERT INTO RSVP (event_id, name, rsvp, comment_id) VALUES (1, 'Tina', 'Yes', 'I am bringing dessert!');

**Comment by noshin7** [comment]
curl -XGET "http://localhost:3000/api/RSVP/1" -H "Content-Type: application/json"

**Pull request merged**